app/src/main/java/app/blef/blef/Game.kt

```
@@ -108,6 +108,7 @@ class Game : AppCompatActivity() {
             queryEngine(
                 R.id.activity_game,
                 baseUrl + if (playerUuid != null)  "/$gameUuid?player_uuid=$playerUuid" else "/$gameUuid",
+                failSilently = true
```

Why fail silently? What if the update endpoint fails (persistently), will the user get any feedback that they can't hope to continue the game?
Same [here](https://github.com/Blef-team/blef_android_app/pull/12/files#diff-e36b69af30fd006fa371260691d6367d9658bf5b1f5cc31bb733b02470de6cd5R127)
and [here](https://github.com/Blef-team/blef_android_app/pull/12/files#diff-202dca539020a4fb35a5dea7f45a8df202c59f0ca969ce885010bc53f8d8ac8dR67)

---

There are a few cases of connection problems:

1. You haven't had internet or the engine has been down since you started
2. You have been able to play a bit, but then either your connection or the engine 100% crashed
3. You have an unstable connection and every third query drops

In case 1, you're sorted. You won't be able to create a game and when you try to continue a game or join via an invite link, the initial engine query will redirect you to the lobby saying it couldn't contact the engine.

In case 2 and 3, we might want to alert the user, but I think we shouldn't do that through the current mechanism. In case 2, you would suddenly start getting an error every second, which is very ennoying. In case 3, you start getting messages every few seconds even though you didn't click anything and most of the seconds nothing changes in the game, which is annoying too. Ideally, for these two cases we would have something like changing the top bar of the app to say '(no connection)' and then unsay it whenever there's a succefful query, but I'd have to see how to do such things.

---

It's case 2 I pointed to 
> will the user get any feedback that they can't hope to continue the game?
No, right?

---

That's true, they won't. 

Though it turns out that we can easily change the title on the top bar. How about, instead of not showing anything in those 'more silent' queries, changing the title on the top bar to 'Blef (no connection)' if the response doesn't come back and 'Blef (connection error)' if it's not successful, and changing it back when it does come back?